src/stripeHelpView.ts

```diff
@@ -12,13 +12,6 @@ export class StripeHelpViewProvider extends StripeTreeViewDataProvider {
     });
     items.push(docsItem);
 
-    const samplesItem = new StripeTreeItem('Find code samples', {
```

Comment:
Can we put `stripe.openSamples` inside the context menu of **Start with a Stripe Sample**? For example, `openDashboardEvent` is available when you right-click an event item when you start streaming events.

Reply:
Sounds good to me